### PR TITLE
[OSDOCS-4610]: Control plane machine sets (4.12 RN)

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.12

Issue:
[OSDOCS-4610](https://issues.redhat.com//browse/OSDOCS-4610), [OSDOCS-4160](https://issues.redhat.com//browse/OSDOCS-4160)

Link to docs preview:
[Control plane machine sets](https://53252--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-mapi-control-plane-machine-sets)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by jeana-redhat** [comment]
FYI @sunzhaohua2 if you are tracking 4.12 feature inclusion in the release notes :slightly_smiling_face:

**Review by elmiko — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -328,6 +328,11 @@ For the OVN-Kubernetes network plug-in, egress firewalls support audit logging u
 [id="ocp-4-12-machine-api"]
 === Machine API
 
+[id="ocp-4-12-mapi-control-plane-machine-sets"]
+==== Control plane machine sets 
+
+{product-title} 4.12 introduces control plane machine sets. Control plane machine sets provide for control plane machines similar management capabilities as compute machine sets provide for compute machines. For more information, see the xref:../machine_management/control_plane_machine_management/cpmso-about.adoc#cpmso-about[Managing control plane machines].
```
Comment (jldohmann) [inline release_notes/ocp-4-12-release-notes.adoc:334]: Just a rewording suggestion that I think makes this a little easier to parse:
```suggestion
{product-title} 4.12 introduces control plane machine sets. Control plane machine sets provide management capabilities for control plane machines that are similar to what compute machine sets provide for compute machines. For more information, see the xref:../machine_management/control_plane_machine_management/cpmso-about.adoc#cpmso-about[Managing control plane machines].
```

**Review by jldohmann — COMMENTED** [review]
LGTM, left a suggestion below 🙂

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (jldohmann) [inline release_notes/ocp-4-12-release-notes.adoc:334]: oh, also:
```suggestion
{product-title} 4.12 introduces control plane machine sets. Control plane machine sets provide for control plane machines similar management capabilities as compute machine sets provide for compute machines. For more information, see xref:../machine_management/control_plane_machine_management/cpmso-about.adoc#cpmso-about[Managing control plane machines].
```

Comment (jeana-redhat) [reply release_notes/ocp-4-12-release-notes.adoc:334]: ty Jesse, I like it :)

**Pull request merged**